File: module5/bai_thi_module_5/ket_noi_database/src/main/java/com/phuc/baithimodule5/model/LoHang.java
package com.phuc.demo.model;

import javax.persistence.*;

@Entity
public class LoHang {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;
    private String maLoHang;
    private Integer soLuong;
    private String ngayNhapHang;
    private String ngaySanXuat;
    private String ngayHetHan;
    @ManyToOne
    @JoinColumn(name = "san_pham", referencedColumnName = "id")
    private SanPham sanPham;

    public LoHang() {
    }


}

File: module5/bai_thi_module_5/baithimodule5/src/app/model/lo-hang.ts
import {SanPham} from './san-pham';

export interface LoHang {
  id: number,
  img:string
  maLoHang: string
  soLuong: number
  ngayNhapHang: string
  ngaySanXuat: string
  ngayHetHan: string
  sanPham: SanPham
}
